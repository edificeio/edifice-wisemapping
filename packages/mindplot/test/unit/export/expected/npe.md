# NIF (NORMAS DE INFORMACIÃ“N FINANCIERA)


