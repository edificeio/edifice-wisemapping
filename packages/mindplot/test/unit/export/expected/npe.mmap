<?xml version="1.0" encoding="UTF-8"?><ap:Map xmlns:ap="http://schemas.mindjet.com/MindManager/Application/2003"><ap:OneTopic><ap:Topic OId="rW54nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/><ap:SubTopicsShape SubTopicsAlignment="urn:mindjet:Center" SubTopicsGrowth="urn:mindjet:Horizontal" SubTopicsGrowthDirection="urn:mindjet:AutomaticHorizontal&#10;                        " VerticalDistanceBetweenSiblings="150"/></ap:Topic></ap:OneTopic><ap:Relationships/></ap:Map>