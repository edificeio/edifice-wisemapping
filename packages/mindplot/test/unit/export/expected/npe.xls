<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Excel.Sheet"?><Workbook xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns:ss="urn:schemas-microsoft-com:office:spreadsheet" xmlns:duss="urn:schemas-microsoft-com:office:dummyspreadsheet" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns="urn:schemas-microsoft-com:office:spreadsheet">
<Styles>
<Style ss:ID="s16" ss:Name="attribute_cell">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
</Style>
<Style ss:ID="s17" ss:Name="attribute_header">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
<Font ss:Bold="1"/>
</Style>
</Styles>
<Worksheet ss:Name="FreeMind Sheet">
<Table>
<Row>
<Cell ss:Index="1">
<Data ss:Type="String"/>
</Cell>
</Row>
</Table>
</Worksheet>
</Workbook>
