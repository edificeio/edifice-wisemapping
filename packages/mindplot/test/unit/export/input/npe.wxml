<map name="72597" version="tango">
    <topic central="-1" text="NIF (NORMAS DE INFORMACIÓN FINANCIERA)" shape="rounded rectangle" id="1"
           fontStyle=";10;#741b47;bold;;" bgColor="#d5a6bd" brColor="#e69138">
        <icon id="object_pencil"/>
    </topic>
    <topic position="6,-284" text="NIF D" shape="rectangle" id="3" fontStyle="Arial;10;;bold;;" bgColor="#509dc0">
        <note>
            <![CDATA[Beneficios a los empleados,impuestos a la utilidad, arrendamientos y capitalización de resultado integral .]]></note>
        <icon id="funy_glasses"/>
        <topic position="211,-280" order="0" text=" Normas aplicables a problemas de determinación de resultados "
               id="19"/>
    </topic>
    <topic position="46,221" text=" CIRCULANTES" shape="rounded rectangle" id="7" fontStyle="Arial;10;;bold;;"
           bgColor="#509dc0">
        <note>
            <![CDATA[Tratamiento contable de los gastos de registro, colocación, unidades de inversión, aplicación supletoria etc.]]></note>
        <icon id="object_phone"/>
        <topic position="236,225" order="0" text="Adquisición temporal de acciones propias" id="22"/>
    </topic>
    <topic position="-143,148" text=" NIF A" shape="rectangle" id="6" fontStyle="Arial;10;;bold;;" bgColor="#509dc0">
        <link url="http://www.youtube.com/watch?v=7YN-sOlkQp0" urlType="url"/>
        <icon id="time_hourglass"/>
        <topic position="-254,152" order="0" text=" Marco conceptual" id="20"/>
    </topic>
    <topic position="-150,-129" text=" NIF C" shape="rectangle" id="4" fontStyle="Arial;10;;bold;;" bgColor="#509dc0">
        <link url="https://sites.google.com/site/contabilidadimcpnif/estructura-de-las-nif" urlType="url"/>
        <icon id="funy_monkey"/>
        <topic position="-366,-125" order="0"
               text="Normas aplicables a conceptos específicos de los estados financieros " shape="line" id="13"/>
    </topic>
    <topic position="183,-153" text="NIF E" shape="rectangle" id="2" fontStyle="Arial;10;;bold;;" bgColor="#509dc0">
        <note><![CDATA[Agricultura y donativos recibidos u otorgados con propósitos no lucrativos.]]></note>
        <icon id="soft_penguin"/>
        <topic position="402,-149" order="0"
               text=" Normas aplicables alas actividades especializadas de distintos sectores" id="18"/>
    </topic>
    <topic position="326,127" text=" NIF B" shape="rectangle" id="5" fontStyle="Arial;10;;bold;;" bgColor="#509dc0">
        <link url="http://www.contaduria.uady.mx/files/cuerpo-acad/caef/aief/resumen_NIF_marco_conceptual.pdf"
              urlType="url"/>
        <icon id="object_house"/>
        <topic position="521,131" order="0" text=" Normas aplicables a los estados financieros en su conjunto" id="21"/>
    </topic>
    <relationship srcTopicId="3" destTopicId="1" lineType="3" endArrow="0" startArrow="0"/>
    <relationship srcTopicId="4" destTopicId="1" lineType="3" endArrow="0" startArrow="0"/>
    <relationship srcTopicId="1" destTopicId="4" lineType="3" endArrow="0" startArrow="0"/>
    <relationship srcTopicId="7" destTopicId="1" lineType="3" endArrow="0" startArrow="0"/>
    <relationship srcTopicId="6" destTopicId="1" lineType="3" endArrow="0" startArrow="0"/>
    <relationship srcTopicId="5" destTopicId="1" lineType="3" endArrow="0" startArrow="0"/>
    <relationship srcTopicId="2" destTopicId="1" lineType="3" endArrow="0" startArrow="0"/>
    <relationship srcTopicId="1" destTopicId="2" lineType="3" endArrow="0" startArrow="0"/>
</map>
