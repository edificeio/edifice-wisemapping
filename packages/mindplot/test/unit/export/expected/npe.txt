1 ✏️ NIF (NORMAS DE INFORMACIÓN FINANCIERA)
2 😎 NIF D
  [Note: Beneficios a los empleados,impuestos a la utilidad, arrendamientos y capitalización de resultado integral .]
	2.1  Normas aplicables a problemas de determinación de resultados 
3 ☎️  CIRCULANTES
  [Note: Tratamiento contable de los gastos de registro, colocación, unidades de inversión, aplicación supletoria etc.]
	3.1 Adquisición temporal de acciones propias
4 ⌛  NIF A
   [Link: http://www.youtube.com/watch?v=7YN-sOlkQp0]
	4.1  Marco conceptual
5 🐵  NIF C
   [Link: https://sites.google.com/site/contabilidadimcpnif/estructura-de-las-nif]
	5.1 Normas aplicables a conceptos específicos de los estados financieros 
6 NIF E
  [Note: Agricultura y donativos recibidos u otorgados con propósitos no lucrativos.]
	6.1  Normas aplicables alas actividades especializadas de distintos sectores
7 🏠  NIF B
   [Link: http://www.contaduria.uady.mx/files/cuerpo-acad/caef/aief/resumen_NIF_marco_conceptual.pdf]
	7.1  Normas aplicables a los estados financieros en su conjunto
